hack/test-update-storage-objects.sh

```diff
@@ -105,12 +105,13 @@ echo "${ETCD_VERSION}" > "${ETCD_DIR}/version.txt"
 # source_file,resource,namespace,name,old_version,new_version
 tests=(
 examples/persistent-volume-provisioning/rbd/rbd-storage-class.yaml,storageclasses,,slow,v1beta1,v1
+hack/testdata/networkpolicy.yaml,networkpolicies,test,allow-from-red-to-blue,v1beta1,v1
```

Comment:
The 3rd space here if for the namespace. Based on the test file, this should be set to default or the test file should add the test namespace.